src/util/tracking.js

```diff
@@ -124,61 +129,66 @@ export const trackApi = log => (response, queryResponses, metadata={}) => {
 	// special case - login requests need to be tracked
 	const loginResponse = queryResponses.find(r => r.login);
 	if ((loginResponse && loginResponse.login.value || {}).member) {
-		const member_id = JSON.stringify(loginResponse.login.value.member.id);
-		trackLogin(log)(response, member_id);
+		const memberId = JSON.stringify(loginResponse.login.value.member.id);
+		trackLogin(log)(response, memberId);
 	}
 	if ('logout' in response.request.query) {
 		trackLogout(log)(response);
 	}
 };
 
-export const trackLogin = log => (response, member_id) =>
+export const trackLogin = log => (response, memberId) =>
 	log(
 		response,
 		{
 			description: 'login',
-			member_id: updateMemberId(response, member_id),
-			track_id_from: response.request.state.track_id,
-			track_id: updateTrackId(response, true),
-			session_id: response.request.state.session_id,
-			url: response.request.info.referrer,
+			memberId: parseInt(updateMemberId(response, memberId), 10) || 0,
+			trackIdFrom: response.request.state[TRACK_ID_COOKIE] || '',
+			trackId: updateTrackId(response, true),
+			sessionId: response.request.state[SESSION_ID_COOKIE],
+			url: response.request.info.referrer || '',
 		}
 	);
 
 export const trackSession = log => response => {
-	if (response.request.state.session_id) {
+	if (response.request.state[SESSION_ID_COOKIE]) {
 		// if there's already a session id, there's nothing to track
 		return null;
 	}
 	return log(
 		response,
 		{
 			description: 'session',
-			member_id: response.request.state.member_id,
-			track_id: updateTrackId(response),
-			session_id: newSessionId(response),
+			memberId: parseInt(response.request.state[MEMBER_ID_COOKIE], 10) || 0,
+			trackId: updateTrackId(response),
+			sessionId: newSessionId(response),
 			url: response.request.url.path,
 		}
 	);
 };
 
-export const logTrack = platform_agent => (response, trackInfo) => {
+export const logTrack = platformAgent => (response, trackInfo) => {
 	const requestHeaders = response.request.headers;
-	const trackLog = {
-		request_id: uuid.v4(),
-		ip: requestHeaders['remote-addr'],
-		agent: requestHeaders['user-agent'],
+	const eventDate = new Date();
+	const record = {
+		timestamp: eventDate.getTime().toString(),
+		requestId: uuid.v4(),
+		ip: requestHeaders['remote-addr'] || '',
+		agent: requestHeaders['user-agent'] || '',
 		platform: 'meetup-web-platform',
-		platform_agent,
+		platformAgent: 'WEB',  // TODO: set this more accurately, using allowed values from avro schema
```

Comment:
yeah after talking with Tom a few weeks ago, I'm not sure if `platformAgent` should always be `WEB` and `platform` should be `mup-web|admin-web|pro-web`, or whether `platform` should always be `meetup-web-platform` and then we could use `platformAgent` to distinguish logs coming from `mup-web|admin-web|pro-web`. I kind of like the latter arrangement because `platform: 'meetup-web-platform'` looks correct and we can treat each consumer as an agent, but that doesn't seem to mesh with current usage, so ¯\_(ツ)_/¯